index.d.ts

```diff
@@ -95,6 +95,37 @@ declare const mem: {
 		options?: mem.Options<ArgumentsType, CacheKeyType, ReturnType>
 	): FunctionToMemoize;
 
+	/**
+	Returns a decorator which memoizes the function provided.
+
+	@example
+	```
+	const mem = require('mem');
+
+	class Example {
+		constructor() {
+			this.i = 0;
+		}
+
+		@mem.mixin();
```

Comment:
From looking at https://www.typescriptlang.org/docs/handbook/decorators.html, it seems decorators can be without the parens. Needs to be tested though.

Reply:
We want to use the return value of `mem.mixin()`, not the function itself.